sdcm/nemesis.py

```diff
@@ -3161,6 +3161,7 @@ def start_and_interrupt_rebuild_streaming(self):
             delay=1
         )
         ParallelObject(objects=[trigger, watcher], timeout=timeout + 60).call_objects()
+        self.target_node.wait_db_up()
```

Comment:
should we add a timeout? i think it is always a good practice to add a timeout (the default one is 3600 seconds, and IMHO it is too big wait)

Reply:
Timeout of 300 seconds added